src/view/pages/FarmerSearch/style.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Style sheet for Farmer Search
+ */
+
+const react = require('react-native');
+const { StyleSheet } = react;
+
+export const styles = StyleSheet.create({
+    name: {
+        fontSize: 20,
+        fontWeight: 'bold',
+    },
```

Comment:
Please use `H1`, `H2`, etc for font size if possible

Reply:
This one seems to be the trickiest. I'm not sure exactly what you mean, but I've changed the sizes to a more 'standard' 24pt and 18pt for name and phone, respectively.

Reply:
So `native-base` has `H1`, `H2`, etc tags that will enforce a specific font size/weight etc. If you could use these instead of giving it a custom font size/weight yourself, that would be ideal because then in the future if we want to change the style of all the text, we can just change these elements. If you have a good reason for not doing so then disregard though